starter-code/javascript/chronometer.js

```diff
@@ -1,23 +1,37 @@
 // Constructor
-// function Chronometer() {
-
-// }
-
-// Chronometer.prototype.startClick = function () {
-
-// };
-
-// Chronometer.prototype.setMinutes = function () {
-  
-// };
-
-// Chronometer.prototype.setSeconds = function () {
-  
-// };
-
-// Chronometer.prototype.twoDigitsNumber = function () {
-  
-// };
+function Chronometer ()  {
+    this.currentTime = 0;
+    this.intervalId = null;
+}
+
+Chronometer.prototype.startClick = function () {
+   var self = this;
+   self.intervalId = window.setInterval(function(){
+    self.currentTime += 1;
+   }, 1000);
+}
+
+ Chronometer.prototype.setMinutes = function () {
+    var self = this;
+    return Math.floor(self.currentTime / 60);
+ };
+
+ Chronometer.prototype.setSeconds = function () {
+    var self = this;
+    self.currentTime = self.currentTime % 60;
+    return self.currentTime;
+ };
+
+ Chronometer.prototype.twoDigitsNumber = function () {
```

Comment:
Be careful, this function only works for minutes. It would be better accept a parameter (let's call it digits) and then you can use it for minutes and seconds. Then, get rid of line 27 and replace minutes by digits.